src/renderer/components/status-bar/status-bar.tsx

```diff
@@ -22,18 +22,24 @@ const NonInjectedStatusBar = observer(({ items }: Dependencies & StatusBarProps)
   const { left, right } = items.get();
 
   return (
-    <div className={styles.StatusBar}>
+    <div className={styles.StatusBar} data-testid="status-bar">
       <div className={styles.leftSide} data-testid="status-bar-left">
         {left.map((Item, index) => (
```

Comment:
Same here, `Item` was capitalized to describe that it is a React.Component (or a class). However, now it's not anymore.